test/support/supportedExtensions.ts

```diff
@@ -0,0 +1,277 @@
+/* tslint:disable max-line-length */
```

Comment:
This is just for isolation purposes. I don't want to depend on an external file that will eventually change its contents. I need the tests to be controlled that's why I duplicated the files.